code/style.css

```diff
@@ -1,4 +1,446 @@
-html {
-  background: #0025ff;
+/****** Media Query Mobile default > 650px ********/
+@import url('https://fonts.googleapis.com/css2?family=Maven+Pro:wght@500&display=swap');
+body{
+  margin: 0;
+  font-family: 'Maven Pro', sans-serif;
+}
+/*******DEFAULT SETTING*****/
+button{
+  padding: 10px; 
+  border:0.16em solid #002e63;
+  background:#002e63;
+  color:white;
+  text-align:center;
+  transition: all 0.20s; /*makes the a smooth "alower" effect on hover*/
+    }
+
+nav button:hover{
+  color:#DDDDDD;
+  border-color:#DDDDDD;
+  }
+
+
+  button:hover{
+  color:#DDDDDD;
+  border-color:#DDDDDD;
+    }
+
+a{
+  text-decoration: none;
+  color:black;
+}
```

Comment:
Indentation is a little off here, the closing tag should not be indented (and there should be a space between the element and the opening tag, _and_ between property key and value):

```
button:hover {
  color: #DDDDDD
  border-color: #DDDDDD
}
```